libs/frontend/feature-render-builder/src/lib/compute-builder-reusable-styles.ts

```diff
@@ -14,10 +14,10 @@ export const computeBuilderReusableStyles = (site: ISite): IRawStyleRecord => {
   const accumulatedBreakpointIds = new Set<string>([DEFAULT_BREAKPOINT_ID])
```

Comment:
I think we're relying on the ordering of the set, which is not good. It's probably better to make this an array, there shouldn't be any downside. Also, it can be moved out of the `appendStyle` for efficiency, since we only need to calculate the `accumulatedBreakpointIds` once per render.

I'll submit a separate PR with these changes to see if everything still works!